packages/flutter/test/material/date_picker_test.dart

```diff
@@ -622,4 +622,72 @@ void _tests() {
 
     semantics.dispose();
   });
+
+
+  testWidgets('chervons animate when scrolling month picker', (WidgetTester tester) async {
+    final Key _datePickerKey = new UniqueKey();
+    DateTime _selectedDate = new DateTime(2016, DateTime.july, 26);
+
+    await tester.pumpWidget(
+      new MaterialApp(
+        home: new StatefulBuilder(
+          builder: (BuildContext context, StateSetter setState) {
+            return new Container(
+              width: 400.0,
+              child: new SingleChildScrollView(
+                child: new Material(
+                  child: new MonthPicker(
+                    firstDate: new DateTime(0),
+                    lastDate: new DateTime(9999),
+                    key: _datePickerKey,
+                    selectedDate: _selectedDate,
+                    onChanged: (DateTime value) {
+                      setState(() {
+                        _selectedDate = value;
+                      });
+                    },
+                  ),
+                ),
+              ),
+            );
+          },
+        ),
+      )
+    );
+
+    // Initial chevron animation state should be dismissed
+    // An AlwaysStoppedAnimation is also found and is ignored
+    find.byType(FadeTransition).evaluate().forEach((Element element) {
```

Comment:
looking for the fade transition seems a bit brittle... i recommend looking for the chevron icon's render object's parent, or some such.

Reply:
How about this:

for(Element element in find.byType(IconButton).evaluate()) {
      final RenderAnimatedOpacity ft = element.ancestorRenderObjectOfType(
        const TypeMatcher<RenderAnimatedOpacity>());
      expect(ft.opacity.value, equals(1.0));
      expect(ft.opacity.status, equals(AnimationStatus.dismissed));
    }